siberi-firebase-support/build.gradle

```diff
@@ -21,7 +21,7 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.mercari:siberi:1.0.0'
+    compile 'com.mercari:siberi:1.0.3'
```

Comment:
we can not set 1.0.3 until we create 1.0.3

Reply:
Exactly. It should be `1.0.2`(the latest published version) or `1.0.0` (as was).  So I'll just revert and let it refer to `1.0.0`, as it can be updated after merging this P-R.